profile/README.md

```diff
@@ -0,0 +1,24 @@
+<p align="center">
+  <img src="https://github.com/ipfs/.github/blob/main/profile/ipfs-powered.png" width="100%" alt="IPFS Powered">
+</p>
+
+The InterPlanetary File System (IPFS) is a set of composable, peer-to-peer protocols for addressing, routing, and transferring [content-addressed data](https://docs.ipfs.tech/concepts/glossary/#content-addressing) in a decentralized file system. Many popular Web3 projects are built on IPFS - see the [ecosystem directory](https://ecosystem.ipfs.tech/) for a list of some of these projects.
+
+Exceptionally talented open communities are building the future of the decentralized web. There are excellent "good first issues" that need your attention and support! Join the movement, get involved today!
+
+### Documentation
+
+- [IPFS Docs](https://docs.ipfs.tech/)
+- [IPFS Specs](https://specs.ipfs.tech/)
+- [Getting Started](https://ipfs.tech/#install)
+- [How IPFS Works](https://ipfs.tech/#how)
+
+### Implementations
+
+- [kubo (go)](https://github.com/ipfs/kubo)
+- [helia (JS)](https://github.com/ipfs/helia)
```

Comment:
I'm not sure whether we want Helia capitalised too. I can see both versions in its' README.

```suggestion
- [Kubo (Go)](https://github.com/ipfs/kubo)
- [Helia (JS)](https://github.com/ipfs/helia)
```